internal/management/state/state.go

```diff
@@ -0,0 +1,25 @@
+package state
```

Comment:
This file looks like a duplicate of `controllers/tempo/internal/management/state/state.go`

Reply:
I moved the file but... seems I didn't do properly in git. Thanks!